README.md

```diff
@@ -48,7 +48,148 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
 )
 ```
 
-## Setting up the accessibility test folder
+## Plugin Setup
+
+There are two different approaches to writing accessibility tests, you may choose the approach that
+suites your team best.
+
+## How to setup automatic accessibility testing
+
+Automatic accessibility testing allows you to write tests via a single spec, it can do this by
+automatically scanning through your projects views for page templates.
+
+### High level of how it works
+* using reflection to discover page templates
+* using the Play app injector to get an instance of each template
+* using scalacheck/magnolia to derive arbitrary values for any template parameters
+* using these arbitrary values to instantiate each page and test it for accessibility
+
+### Which testing approach is best for your team
+#### Manual accessibility testing
+* Teams might already have the linter and some coverage via their view unit tests, and may want to continue down that route.
+#### Automatic accessibility testing
+* Teams adopting the linter for the first time might find it easier to go with the automated route.
+
+1. Copy the template spec below into your project
+
+```scala
+import config.AppConfig
+import controllers.{ContactForm, ContactHmrcForm}
+import model.{AccessibilityForm, FeedbackForm, ReportProblemForm, SurveyForm}
+import org.scalacheck.Arbitrary
+import org.scalatest.Reporter
+import org.scalatest.events.{Event, SuiteCompleted, TestPending}
+import play.api.data.Form
+import play.api.data.Forms._
+import play.twirl.api.Html
+import uk.gov.hmrc.scalatestaccessibilitylinter.views.AutomaticAccessibilitySpec
+import views.html._
+import scala.collection.mutable.ListBuffer
```

Comment:
these imports need cleaning up - maybe this, to cover example code and significant things referenced in comments?
```suggestion
import org.scalacheck.Arbitrary
import play.api.data.Form
import play.twirl.api.Html
import uk.gov.hmrc.scalatestaccessibilitylinter.views.AutomaticAccessibilitySpec
import views.html._
```